opensquirrel/utils/bit_register_mapping.py

```diff
@@ -0,0 +1,73 @@
+"""This module contains the ``BitStringMapping`` class."""
+
+from __future__ import annotations
+
+from collections import defaultdict
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from opensquirrel.ir import Measure
+
+
+class BitStringMapping:
+    """Utility class to create bitstringmappings for opensquirrel exporter passes."""
+
+    def __init__(self) -> None:
+        """Init of the ``BitStringMapping``."""
+        self._mapping: defaultdict[int, list[int]] = defaultdict(list)
+        self._bit_allocation: dict[int, int] = {}
```

Comment:
Maybe useful to define, at least, aliases for these mapping and bit allocation types?